s2a/src/test/resources/client.csr

```diff
@@ -0,0 +1,16 @@
+-----BEGIN CERTIFICATE REQUEST-----
```

Comment:
It is weird to commit the csr. That is temporary, right?

Reply:
From the [README](https://github.com/grpc/grpc-java/blob/master/s2a/src/test/resources/README.md),the csr is created as part of the steps to regenerate the certs in this folder, which have a lifetime of 20 yrs. This resources directory is similar to the one we have the S2A Go client: https://github.com/google/s2a-go/tree/main/testdata